packages/patternfly-4/react-inline-edit-extension/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@patternfly/react-inline-edit-extension",
+  "version": "1.0.0",
+  "description": "This library provides inline editing support for PatternFly 4 React table",
+  "main": "dist/js/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/js/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public",
+    "tag": "prerelease"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/patternfly/patternfly-react.git"
+  },
+  "keywords": [
+    "react",
+    "patternfly",
+    "table",
+    "reacttabular"
+  ],
+  "author": "Red Hat",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/patternfly/patternfly-react/issues"
+  },
+  "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/",
+  "dependencies": {
+    "@patternfly/patternfly": "1.0.240",
```

Comment:
needs updating now (just noticed this by doing `yarn install` and seeing yarn lock diff):
`"@patternfly/patternfly": "1.0.244",`